pyrenew_hew/pyrenew_hew_model.py

```diff
@@ -227,61 +234,89 @@ def sample(
             concentration_rv=self.ed_neg_bin_concentration_rv,
         )
 
-        observed_ed_visits = ed_visit_obs_rv(
+        sampled_ed_visits = ed_visit_obs_rv(
```

Comment:
```suggestion
        observed_ed_visits = ed_visit_obs_rv(
```

also change the return

Reply:
Done in c108adf